### PR TITLE
Feature/538 update project muse onix export

### PR DESCRIPTION
**Pull request by brendan-oconnell**
https://github.com/thoth-pub/thoth/issues/538

### CODE REVIEW

**Review by rhigman — CHANGES_REQUESTED** [review]
Could also add the `EditionStatement` tag (between contributors and languages) - awkward that they apparently don't support `EditionNumber`, but there's logic for constructing the verbal statement [in the MARC export](https://github.com/thoth-pub/thoth/blob/a178ea883540bcb99a2d6b9af450f0ccfcb5fca0/thoth-export-server/src/marc21/marc21record_thoth.rs#L216).

Looks good overall, thanks!

File: thoth-export-server/src/xml/onix3_project_muse.rs [hunk 1]
```diff
@@ -204,8 +207,10 @@ impl XmlElementBlock<Onix3ProjectMuse> for Work {
                         })?;
                     }
                     for subject in &self.subjects {
-                        // Project MUSE can't process records containing keywords
-                        if subject.subject_type != SubjectType::KEYWORD {
+                        // According to spec, Project MUSE only accepts BIC and BISAC subject codes
```
[the remark this answers is not in the document]
Comment (brendan-oconnell) [reply thoth-export-server/src/xml/onix3_project_muse.rs:210]: Yes, I think you're right - no need to remove them in this case.

File: thoth-export-server/src/xml/onix3_project_muse.rs [hunk 2]
```diff
@@ -372,7 +396,7 @@ impl XmlElementBlock<Onix3ProjectMuse> for Work {
                         supplies.insert(
                             landing_page.to_string(),
                             (
-                                "01".to_string(),
+                                "02".to_string(),
```
[the remark this answers is not in the document]
Comment (brendan-oconnell) [reply thoth-export-server/src/xml/onix3_project_muse.rs:399]: Yeah, I also noticed that many other outputs use 01, but I agree that 02 is generally going to be more appropriate - and in this case, follows Project MUSE's spec. I agree that it seems like a good idea to fix this in the other outputs.

**Comment by brendan-oconnell** [comment]
> Could also add the `EditionStatement` tag (between contributors and languages) - awkward that they apparently don't support `EditionNumber`, but there's logic for constructing the verbal statement [in the MARC export](https://github.com/thoth-pub/thoth/blob/a178ea883540bcb99a2d6b9af450f0ccfcb5fca0/thoth-export-server/src/marc21/marc21record_thoth.rs#L216).
> 
> Looks good overall, thanks!

Thanks! I agree that it's weird that we can't just put in `EditionNumber`, but the MARC logic was helpful for `EditionStatement`, thanks for that!

**Review by rhigman — APPROVED** [review]

File: thoth-export-server/src/xml/onix3_project_muse.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (ja573) [reply thoth-export-server/src/xml/onix3_project_muse.rs:420]: We based the majority of the ONIX outputs on OAPEN's, so now that they've confirmed they're happy with us changing the code to 02 let's update it in the rest and look out for any potential complains

[the remark this answers is not in the document]
Comment (brendan-oconnell) [reply thoth-export-server/src/xml/onix3_project_muse.rs:420]: Great, I'll create an issue for this.

**Pull request merged**